certified-connectors/Tesseron Invoice/apiDefinition.swagger.json

```diff
@@ -0,0 +1,245 @@
+            "properties": {
+              "dateFrom": {
+                "type": "string",
+                "description": "Specify the start date",
+                "title": "Date from"
+              },
+              "dateTo": {
+                "type": "string",
+                "description": "Specify the end date",
+                "title": "Date to"
+              },
+              "pause": {
+                "type": "string",
+                "description": "Specify the pause",
+                "title": "Pause"
+              },
+              "noInvoice": {
+                "type": "boolean",
+                "description": "Specify whether the position note is not billable",
+                "title": "No Invoice",
+                "enum": [
+                  true,
+                  false
+                ],
+                "default": false
+              },
+              "extraCharge": {
+                "type": "boolean",
+                "description": "Specify whether surcharges are calculated",
+                "title": "Extra Charge",
+                "enum": [
+                  true,
+                  false
+                ],
+                "default": false
+              },
+              "hint": {
+                "type": "string",
+                "description": "Specify a hint for your position note",
+                "title": "Hint"
+              },
+              "username": {
+                "type": "string",
+                "description": "Specify a user name",
+                "title": "Username"
+              },
+              "ticketID": {
+                "type": "integer",
+                "description": "Specify a ticket id",
+                "title": "Ticket ID"
+              },
+              "serviceContractID": {
+                "type": "integer",
+                "description": "Specify a service contract id",
+                "title": "Service Contract ID"
+              }
+            },
+            "required": [
+              "dateFrom",
+              "dateTo",
+              "noInvoice",
+              "extraCharge"
+            ]
+          }
+        }
+      ]
+    },
+    "/CreateActivityRecording": {
+      "post": {
+        "responses": {
+          "default": {
+            "description": "default"
+          },
+          "200": {
+            "description": "success",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "Message": {
+                  "type": "string",
+                  "description": "Message"
+                },
+                "Success": {
+                  "type": "boolean",
+                  "description": "Success"
+                },
+                "ActivityRecordingId": {
+                  "type": "string",
+                  "description": "Activity Recording ID"
+                }
+              }
+            }
+          }
+        }
+      },
+      "summary": "Activity Recording ID",
```

Comment:
@developmentLL the operation properties should be within the scope of post action. before or after the responses.

Please fix all operations in the swagger file.